chaoscenter/authentication/api/handlers/rest/project_handler.go

```diff
@@ -796,3 +882,50 @@ func GetProjectRole(service services.ApplicationService) gin.HandlerFunc {
 
 	}
 }
+	
+// DeleteProject  		godoc
+//
+//	@Description	Delete a project.
+//	@Tags			ProjectRouter
+//	@Accept			json
+//	@Produce		json
+//	@Failure		400	{object}	response.ErrProjectNotFound
+//	@Failure		500	{object}	response.ErrServerError
+//	@Success		200	{object}	response.Response{}
+//	@Router			/delete_project [post]
+//
+// DeleteProject is used to delete a project.
+func DeleteProject (service services.ApplicationService) gin.HandlerFunc {
+	return func(c *gin.Context) {
+		var deleteProjectRequest entities.DeleteProjectInput
+		err := c.BindJSON(&deleteProjectRequest)
```

Comment:
If delete_project takes just one parameter i.e project_id, better to take it from the URL itself instead of taking it from the request body. `/delete_project/{project_id}`

Reply:
on it.